src/components/Buttons/FutureButton/FutureButton.tsx

```diff
@@ -35,7 +35,7 @@ const FutureButton: React.FC<FutureButtonProps> = ({
 	glow,
 	loading,
 	alt,
-  disabled = false,
+	disabled = false,
```

Comment:
Not sure why it was happened. Husky didn't work with this file? Hmm, not sure yet.

Reply:
No clue to be honest!